### PR TITLE
(PC-32822) feat(NC): Remove euro symbols in code (PC-32820) (PC-32823)

### PR DESCRIPTION
**Pull request by lbeneston-pass**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-32822

## Flakiness

If I had to re-run tests in the CI due to flakiness, I add the incident [on Notion][2]

## Checklist

I have:

- [x] Made sure my feature is working on web.
- [x] Made sure my feature is working on mobile (depending on relevance : real or virtual devices)
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

## Screenshots

**delete** _if no UI change_

| URLs          | Screenshot | User |
| :---------- | :----------: |  :----------: |
| http://localhost:5173/notices-legales            | <details close><summary>Illustration</summary>Capture d’écran 2024-11-07 à 10 03 16</details>  |  Déconnecté     |
| http://localhost:5173/anniversaire-18-ans)           |  <details close><summary>Illustration</summary>Capture d’écran 2024-11-07 à 10 07 48</details> | Déconnecté       |
| http://localhost:5173/selection-age/eligible?age=15  |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 11 48 29 </details> | mobile       |
| http://localhost:5173/selection-age/eligible?age=16  |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 11 48 56 </details> | mobile       |
| http://localhost:5173/selection-age/eligible?age=17  |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 11 49 06  </details> | mobile       |
| http://localhost:5173/selection-age/eligible?age=18  |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 11 49 15 </details> | mobile       |
| http://localhost:5173/tutoriel/selection-age/eligible?age=15            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 11 59 01 </details> |   Déconnecté      |
| http://localhost:5173/tutoriel/selection-age/eligible?age=16            |  <details close><summary>Illustration</summary>Capture d’écran 2024-11-07 à 12 00 54  </details> |    Déconnecté    |
| http://localhost:5173/tutoriel/selection-age/eligible?age=17            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 12 01 32 </details> |     Déconnecté   |
| http://localhost:5173/tutoriel/selection-age/eligible?age=18            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 12 02 15 </details> |   Déconnecté     |
| http://localhost:5173/creation-compte/confirmation-beneficiaire            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 12 06 11 </details> |  Bénéficiaire 17 ans      |
| http://localhost:5173/profil            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 12 08 06  Capture d’écran 2024-11-07 à 12 11 14 </details> |   Bénéficiaire     |
|http://localhost:5173/offre/1660            |  <details close><summary>Illustration</summary>Capture d’écran 2024-11-07 à 12 20 50 Capture d’écran 2024-11-07 à 15 00 21 Capture d’écran 2024-11-07 à 15 05 54 Capture d’écran 2024-11-07 à 15 00 11 Capture d’écran 2024-11-07 à 14 59 18Capture d’écran 2024-11-07 à 15 09 16 </details> |   Bénéficiaire    |
| http://localhost:5173/anniversaire-18-ans            |   <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 14 54 07 </details>  |  Cheatcode > Eighteen Birthday      |
| http://localhost:5173/recharge-credit-anniversaire          |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 14 56 59 </details> |  Cheatcode > Notification rechargement crédit      |
| http://localhost:5173/reservations            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 15 11 21Capture d’écran 2024-11-07 à 15 12 43 </details> | Bénéficiaire avec une résa       |
| http://localhost:5173/favoris            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 15 14 03 </details> |  Bénéficiaire avec favoris      |
|  http://localhost:5173/accueil            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 15 15 57 </details> | Bénéficiaire       |
| http://localhost:5173/recherche            |  <details close><summary>Illustration</summary> Capture d’écran 2024-11-07 à 15 26 19 Capture d’écran 2024-11-07 à 15 26 04 Capture d’écran 2024-11-07 à 15 25 53 Capture d’écran 2024-11-07 à 15 25 41 </details> |    Recherche par prix    |

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a
[2]: https://www.notion.so/passcultureapp/cb45383351b44723a6f2d9e1481ad6bb?v=10fe47258701423985aa7d25bb04cfee&pvs=4

### CODE REVIEW

File: __snapshots__/features/tutorial/pages/onboarding/OnboardingAgeInformation.native.test.tsx.native-snap [hunk 1]
```diff
@@ -6036,7 +6039,8 @@ exports[`OnboardingAgeInformation should render correctly for 18-year-old 1`] =
                       ]
                     }
                   >
-                    Tu auras 2 ans pour utiliser tes 300 €
```
Comment (cgerrard-pass) [inline __snapshots__/features/tutorial/pages/onboarding/OnboardingAgeInformation.native.test.tsx.native-snap:6039]: Pourquoi il y a un saut à la ligne maintenant?

File: src/libs/parsers/getDisplayPrice.ts [hunk 2]
```diff
@@ -22,18 +22,18 @@ export const formatToFrenchDecimal = (priceInCents: number, options?: FormatPric
   return formatter.format(euros)
 }
 
-export const formatPriceInEuroToDisplayPrice = (priceInEuro: number) =>
-  formatToFrenchDecimal(convertEuroToCents(priceInEuro))
+export const parseCurrency = (priceInEuro: number) =>
```
Comment (cgerrard-pass) [inline src/libs/parsers/getDisplayPrice.ts:25]: Je dis ça sans conviction, mais ça n'aurait pas été plus simple de d'avoir que parseCurrencyFromCents et de toujours specifier le nombre de centimes dans tout l'app? On a moins de chance de se tromper s'il n'y a qu'une manière de mettre le prix

**Review by cgerrard-pass — APPROVED** [review]

File: __snapshots__/features/tutorial/pages/onboarding/OnboardingAgeInformation.native.test.tsx.native-snap (hunk 1 above)
Comment (lbeneston-pass) [reply __snapshots__/features/tutorial/pages/onboarding/OnboardingAgeInformation.native.test.tsx.native-snap:6039]: Je ne sais pas mais c'est juste dans les snapshots. Je pense que si tu as un élément textuel dans des {} il fait ça ?

File: src/libs/parsers/getDisplayPrice.ts (hunk 2 above)
Comment (lbeneston-pass) [reply src/libs/parsers/getDisplayPrice.ts:25]: Si, c'est une amélioration que j'aimerais faire prochainement ! Je vais créer un ticket

Comment (lbeneston-pass) [reply src/libs/parsers/getDisplayPrice.ts:25]: Ticket : https://passculture.atlassian.net/browse/PC-32912

**Pull request merged**